recipes/libtiff/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.0.6" %}
+
+package:
+    name: libtiff
+    version: {{ version }}
+
+source:
+    fn: tiff-{{ version }}.tar.gz
+    url: ftp://ftp.remotesensing.org/pub/libtiff/tiff-{{ version }}.tar.gz
+    md5: d1d2e940dea0b5ad435f21f03d96dd72
+    patches:
+        - def_snprintf.patch  # [win]
```

Comment:
Yeah, I'm pretty sure it is necessary was just trying to track down if this has reached upstream yet. Sounds like there are at least aware of the problem and we seem to be using the same patch others are.